pkg/jobrunaggregator/jobrunaggregatoranalyzer/pass_fail.go

```diff
@@ -246,7 +321,11 @@ func testShouldAlwaysPass(name string) bool {
 		case strings.HasSuffix(name, "-openshift-e2e-test container test"):
 			return true
 		}
+	} else if jobrunaggregatorlib.IsDisruptionTest(name) {
+		// For disruption tests, we'll calculate test/fail later based on actual durations.
+		return true
```

Comment:
this should never be called right?  If it should never be called, let's return some crazy message about never being here and force a failure so we notice.

Reply:
Leftover from a previous approach, removed.